integrated_channels/sap_success_factors/migrations/0004_auto_20220324_1550_squashed_0006_auto_20220330_1157.py

```diff
@@ -0,0 +1,67 @@
+# Generated by Django 3.2.12 on 2022-03-30 14:39
+
+from django.db import migrations, models
+import django.utils.timezone
+import model_utils.fields
+
+
+class Migration(migrations.Migration):
+
+    replaces = [('sap_success_factors', '0004_auto_20220324_1550'), ('sap_success_factors', '0005_auto_20220325_1757'), ('sap_success_factors', '0006_auto_20220330_1157')]
+
+    dependencies = [
+        ('sap_success_factors', '0003_alter_sapsuccessfactorslearnerdatatransmissionaudit_completed_timestamp'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='sapsuccessfactorslearnerdatatransmissionaudit',
+            name='enterprise_customer_uuid',
+            field=models.UUIDField(blank=True, null=True),
+        ),
+        migrations.AddField(
+            model_name='sapsuccessfactorslearnerdatatransmissionaudit',
+            name='modified',
+            field=model_utils.fields.AutoLastModifiedField(default=django.utils.timezone.now, editable=False, verbose_name='modified'),
+        ),
+        migrations.AlterField(
+            model_name='sapsuccessfactorslearnerdatatransmissionaudit',
+            name='created',
+            field=model_utils.fields.AutoCreatedField(default=django.utils.timezone.now, editable=False, verbose_name='created'),
+        ),
+        migrations.AlterField(
+            model_name='sapsuccessfactorslearnerdatatransmissionaudit',
+            name='enterprise_course_enrollment_id',
+            field=models.IntegerField(blank=True, db_index=True, null=True),
+        ),
+        migrations.AlterField(
+            model_name='sapsuccessfactorslearnerdatatransmissionaudit',
+            name='error_message',
+            field=models.TextField(blank=True, null=True),
+        ),
+        migrations.AlterField(
+            model_name='sapsuccessfactorslearnerdatatransmissionaudit',
+            name='status',
+            field=models.CharField(blank=True, max_length=100, null=True),
+        ),
+        migrations.AddField(
+            model_name='sapsuccessfactorslearnerdatatransmissionaudit',
+            name='subsection_id',
+            field=models.CharField(blank=True, db_index=True, max_length=255, null=True),
+        ),
+        migrations.AddField(
+            model_name='sapsuccessfactorslearnerdatatransmissionaudit',
+            name='subsection_name',
+            field=models.CharField(max_length=255, null=True),
+        ),
+        migrations.AlterField(
+            model_name='sapsuccessfactorslearnerdatatransmissionaudit',
+            name='enterprise_course_enrollment_id',
+            field=models.IntegerField(blank=True, db_index=True, null=True),
+        ),
```

Comment:
I think this is a repeat of the earlier AlterField, which may be from your manual change? You only need one if they are the same, and I'm wondering if this was a new field in these migrations or not.

Reply:
This wasn't a new field, it was previously there which we fixed earlier but was removed here so `AlterField` appeared in `0004` migration

https://github.com/openedx/edx-enterprise/commit/b0e418101bfb1b0bbc5188b517adfa58547dbf11#diff-6f4694ec4c254f702264a1cd1e0c2bc82e8914e39d4ec9778a743f47bd00a4c2L277